Libraries/LLM/Load.swift

```diff
@@ -15,6 +15,7 @@ struct LLMError: Error {
 /// Load and return the model and tokenizer
 public func load(
     hub: HubApi = HubApi(), configuration: ModelConfiguration,
+    overrideWeights: URL? = nil,
```

Comment:
Override URL for weights -- useful if we want to use fused weights from LoRA

Reply:
Here is the fuse command:

- https://github.com/ml-explore/mlx-swift-examples/pull/46/files#diff-57fb3aa82dee5fb24ff29e9b450e9745821d31bb65cf126cb27fb4e0d4c051deR150

So you would make a new Hub entry with the old config / tokenizer + new weights?  Do you upload that to hugging face or just do that locally?

I was just putting the fused weights e.g. in /tmp and reading the original model files + new weights that way.  It looks like the `Hub` API from `swift-transformers` doesn't provide methods to upload so I would need to see how that is done (if we need the upload part).

Reply:
> So you would make a new Hub entry with the old config / tokenizer + new weights? 

Yes but it's not a Hub entry it's just a local path (the code can use HF repos or local paths)

> Do you upload that to hugging face or just do that locally?

Both are supported. But I think it's fine to just do it locally for now.

Reply:
OK, I will set this up!

Reply:
Done -- I will summarize in just a sec.